doc/tutorial/fcm_specifics

```diff
@@ -0,0 +1,1484 @@
+                <br/>
+                <code>fcm resolve --accept working &lt;FILENAME&gt;</code>
+                <br/>
+                <pre>
+cd ~/my_fcm_tutorial/specifics/branch_resolve
+fcm bls
+<font color="blue">
+[info] fcm:fcm_spe@78: 9 match(es)
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_conflict_tree@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_conflict_tree2@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_merge_custom@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_merge_custom2@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_propdel@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_propedit@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_propset_exe@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_propset_ign@78
+fcm:fcm_spe/branches/dev/$LOGNAME/branch_resolve@78
+</font>
+fcm merge fcm:fcm_spe_br/dev/$LOGNAME/branch_conflict_tree
+<font color="blue">
+Eligible merge(s) from /specifics/branches/dev/$LOGNAME/branch_conflict_tree@78: 46 45
+Enter a revision (or just press &lt;return&gt; for "46"): 
+--------------------------------------------------------------------------------
+Merge: /specifics/branches/dev/$LOGNAME/branch_conflict_tree@46
+ c.f.: /specifics/trunk@643
+-------------------------------------------------------------------------dry-run
+--- Merging r44 through r46 into '.':
+   C src/utility
+   A src/utility/tool_list.txt
+D    tree_conflict.txt
+Summary of conflicts:
+  Tree conflicts: 1
+-------------------------------------------------------------------------dry-run
+Would you like to go ahead with the merge?
+Enter "y" or "n" (or just press &lt;return&gt; for "n"): <font color="black">y</font>
+Merge succeeded.
+--------------------------------------------------------------------------actual
+--- Recording mergeinfo for merge of r42 through r43 into '.':
+ U   .
+--- Merging r44 through r46 into '.':
+   C src/utility
+   A src/utility/tool_list.txt
+D    tree_conflict.txt
+--- Recording mergeinfo for merge of r44 through r46 into '.':
+ U   src/utility/tool_list.txt
+ G   .
+Summary of conflicts:
+  Tree conflicts: 1
+--------------------------------------------------------------------------actual
+</font>
+fcm st
+<font color="blue">
+ M      .
+      C src/utility
+      >   local dir obstruction, incoming dir add upon merge
+ M      src/utility/tool_list.txt
+D       tree_conflict.txt
+Summary of conflicts:
+  Tree conflicts: 1
+</font>
+fcm resolve --accept working src/utility
+<font color="blue">
+Resolved conflicted state of 'src/utility'
+</font>
+</pre>
+                <br/>
+                The conflict has been resolved.
+                <br/>
+                <br/>
+                We will use this command further in the <a href="#S5">tree conflicts</a> 
+                sub-section.
+            </div>
+            <div class="well">
+                <h4>Tree conflicts</h4>
+                Tree conflict, which are otherwise known as a structural or filesystem-based 
```

Comment:
Tree conflict*s*, otherwise known ... conflicts can be more complicated tha*n* text conflicts.